src/api/supabase/queries/cases.ts

```diff
@@ -2,13 +2,20 @@ import { UUID } from 'crypto';
 import supabase from '../createClient';
 
 export async function getAllCases() {
-  const { data, error } = await supabase.from('cases').select();
+  const { data, error } = await supabase
+    .from('cases')
+    .select()
+    .or('needs_interpreter.eq.true,needs_attorney.eq.true');
   if (error) throw new Error(`Error fetching all cases: ${error.message}`);
   return data;
 }
 
 export async function getNCases(n: number) {
-  const { data, error } = await supabase.from('cases').select().limit(n);
+  const { data, error } = await supabase
+    .from('cases')
+    .select()
+    .or('needs_interpreter.eq.true,needs_attorney.eq.true')
+    .limit(n);
   if (error) throw new Error(`Error fetching ${n} cases: ${error.message}`);
   return data;
 }
```

Comment:
we might need to update these after the switch to using the database function for the queries. May or may not also need to update the database func?